src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -265,6 +265,10 @@ function getCreateOrUpdateSchemasSQL(newAddedAppIdList: string[], props: Resourc
       table_event_parameter: odsTableNames.event_parameter,
```

Comment:
Should we remove these legacy tables?

Reply:
sp-migrate-ods-events-1.0-to-1.1.sql still use it.

Reply:
refactored it